src/catkin_pkg/package_templates.py

```diff
@@ -436,4 +495,8 @@ def get_person_tag(tagname, person):
 
     temp_dict['components'] = package_template.catkin_deps
 
-    return ctemp.substitute(temp_dict)
+    temp_dict['build_type'] = ''.join([])
+    temp_dict['architecture_independent'] = ''.join([])
+    temp_dict['deprecated'] = ''.join([])
```

Comment:
Since these three value are never being set please removed them here and also remove the placeholders from the template.